src/test/java/net/sf/jabref/logic/l10n/LocalizationConsistencyTest.java

```diff
@@ -172,7 +172,7 @@ public void findObsoleteMenuLocalizationKeys() throws IOException {
             System.out.println(obsoleteKeys.stream().map(Object::toString).collect(Collectors.joining("\n")));
             System.out.println();
             System.out.println("1. REMOVE THESE FROM THE ENGLISH LANGUAGE FILE");
-            System.out.println("3. EXECUTE gradlew update");
+            System.out.println("2. EXECUTE gradlew localizationUpdate");
```

Comment:
If you already touch the localization tests, could you also remove the `System.out.println` statements by moving everything to the error message.
So for example this here should be 
`assertEquals("Obsolte kyes found in menu properties file. \n 1. Remove these... \n 2...", emptyList, obsolteKeys)`